internal/pkg/cli/pipeline_init.go

```diff
@@ -59,7 +59,7 @@ type initPipelineVars struct {
 	environments      []string
 	githubOwner       string
 	githubRepo        string
-	githubURL         string
+	URL               string
```

Comment:
changing this from `githubURL` to `URL` has the side effect of exporting only this struct member, which may not be desired behavior. Can we change it to `providerURL` or just `url`? I know that typically we want to render acronyms in all caps but this warrants an exception.

Reply:
Thanks! Changed it to `repoURL`.